client-app/ui-kit/components/molecules/image-gallery/vc-image-gallery.vue

```diff
@@ -1,14 +1,37 @@
 <template>
   <div
     ref="mainImageElement"
-    class="square relative flex flex-col items-center justify-center overflow-hidden rounded-sm border border-gray-100"
+    class="vc-image-gallery square relative flex flex-col overflow-hidden rounded-sm border border-gray-100"
+    data-te-lightbox-init
   >
+    <template v-if="images.length > 1">
+      <VcImage
+        v-for="(image, i) in images"
+        :key="image.url || i"
+        :src="image.url"
+        size-suffix="md"
+        class="vc-image-gallery__main-image absolute top-0 h-full w-full cursor-zoom-in rounded-sm object-cover object-center data-[te-lightbox-disabled]:cursor-auto"
```

Comment:
Don't mix BEM with style helpers, if you use BEM, then move all applied styles to class via `@apply`